Package.swift

```diff
@@ -90,6 +91,19 @@ let package = Package(
       ]
     ),
 
+    .target(
+      name: "AssertionDependency",
```

Comment:
We need to `@_exported import AssertDependency` in the [`Exports.swift`](https://github.com/tgrapperon/swift-dependencies-additions/blob/main/Sources/DependenciesAdditions/Exports.swift) umbrella too.